client/ayon_traypublisher/plugins/create/create_editorial_advanced.py

```diff
@@ -0,0 +1,1319 @@
+
+            matched_product_items = []
+            for root, foldernames, filenames in os.walk(abs_clip_folder):
+                # iterate all product names in enabled presets
+                for pres_product_data in product_type_presets:
+                    product_name = pres_product_data["product_name"]
+
+                    product_data_base = {
+                        "preset_name": product_name,
+                        "clip_dir_subpath": "",
+                        "product_name": product_name,
+                        "files": [],
+                    }
+                    root = root.replace("\\", "/")
+                    cl_part_path = root.replace(abs_clip_folder, "")
+
+                    for folder in foldernames:
+                        product_data = deepcopy(product_data_base)
+                        # need to include more since variants might occure
+                        pattern_search = re.compile(
+                            f".*({re.escape(product_name)}{VARIANTS_PATTERN}).*"
+                        )
+                        match = pattern_search.search(folder)
+                        if not match:
+                            continue
+
+                        # form partial path without starting slash
+                        partial_path = os.path.join(
+                            cl_part_path.lstrip("/"), folder
+                        ).replace("\\", "/")
+
+                        # update product data it will be deepcopied later
+                        # later in files processor
+                        product_data.update(
+                            {
+                                "product_name": match.group(0),
+                                "clip_dir_subpath": partial_path,
+                            }
+                        )
+                        nested_files = list(
+                                os.listdir(os.path.join(root, folder)))
+                        self._include_files_for_processing(
+                            product_name,
+                            nested_files,
+                            product_data,
+                            matched_product_items,
+                            strict=False,
+                        )
+
+                    product_data_base["clip_dir_subpath"] = "/"
+                    self._include_files_for_processing(
+                        product_name,
+                        filenames,
+                        product_data_base,
+                        matched_product_items,
+                    )
+
+                # No matching product data can be skipped
+                if not matched_product_items:
+                    self.log.warning(
+                        f"No matching product data found in {root}."
+                        " Skipping folder."
+                    )
+                    continue
+
+                clip_content[clip_folder.replace(media_folder_path, "")] = (
```

Comment:
`clip_folder` is folder name, it does not contain `media_folder_path`
```suggestion
                clip_content[clip_folder] = (
```